### DISCUSSION TITLE
Estimated Amount using 835 - No CPT/ HCPCS

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#115] [Q&A] [answered]
Hi There - Can you provide guidance on what is an appropriate response for the estimated amount (when the % of BC is the reason) and there is no CPT or HCPCS for that charge line? I have many charge lines for drugs on rev code 250 where there is no code to tie to on the 835. Can I encode 999999999 (nine 9s) in the estimated amount field and "Unable to calculate estimated amount as the 835 does not list a CPT/HCPCS" in the additional payor note field? Thanks for the guidance

**Comment by RidenourK** [comment, marked as answer]
@vduffel99 Thank you for your question. 

At 45 CFR § 180.20 we defined “estimated allowed amount” as the average dollar amount that the hospital has historically received from a third-party payer for an item or service when the payer-specific negotiated charge for that item or service can only be expressed as a percent or algorithm. In the CY 2024 OPPS/ASC Final Rule we stated that **we believed hospitals should retain flexibility, in the interest of reducing burden, to determine the best data source for calculating the estimated allowed amount.**
